FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -2,52 +2,35 @@
 {% load i18n %}
 
 {% block heading %}
-    Students
+    {# Translators: Student page header #}
+    {% trans "Students" %}
 {% endblock %}
 
-{% block content %}
-<head>
-<style>
-table {
-    border-collapse: collapse;
-    width: 50%;
-}
-
-td, th {
-    border: 1px solid #dddddd;
-    text-align: left;
-    padding: 8px;
-}
-
-tr:nth-child(even) {
-    background-color: #dddddd;
-}
 
-</style>
-</head>
+{% block content %}
 <body>
-
-
-{% if student_list %}
-    <div>
-    <table align = "center">
-     <tr>
-      <th>CCID</th>
-      <th>Year of Study</th>
-      <th>Program</th>
-     </tr>
-    {% for student in student_list %}
-        <tr>
-         <td>{{ student.ccid }}</td>
-         <td>{{ student.year }}</td>
-         <td>{{ student.program }}</td>
-         <td><a style="color:blue;" href="/student/{{ student.ccid }}/">View</a></td>
-        </tr>
-    {% endfor %}
-    </table>
-    </div>
-{% else %}
-    <p>No students are available.</p>
-{% endif %}
+<div>
+<table align = "center">
```

Comment:
The changes added to the CSS are not actually reflected on this template once it renders. Interesting since the css is linked on the base.html template and DO work on the base_with_banner html but aren't appearing on this template. It leads the page to look really odd though.

Reply:
Browser cache, thanks for that @jrforest nevermind this comment.